tests/test_game.py

```diff
@@ -24,20 +23,6 @@ def test_default_state(self):
         self.assertEqual(self.robot.zone, 0)
         self.assertEqual(self.robot.mode, GameMode.DEVELOPMENT)
 
-    def test_game_state(self):
-        self.mock.new_gamestate()
-        zone = 2
-        self.thread = Thread(target=zone_script.poll, args=("/tmp/robotd/", zone, self.stop_event))
```

Comment:
I think this also makes some of the imports and instance attributes redundant (including the `tearDown` method)